### ISSUE TITLE
Error when trying to install spacy-3.0.6.tar.gz

### ISSUE THREAD

**Issue by YamiZee** [daswer123/xtts-api-server#77]
I'm trying to set this up according to [this](https://docs.sillytavern.app/extensions/xtts/) guide. No matter what python version I use or if it's with conda or the standard venv, the installation always fails when trying to install spacy-3.0.6.tar.gz. 

Many of the errors complain:
`Variables cannot be declared with 'cpdef'. Use 'cdef' instead.` 

And at the end:
```
× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

I also find it strange that the installer tries to install every version of spacy, starting with spacy-3.7.2 and going down all the way to spacy-3.0.6 where it bugs out.

**Comment by jgilbertfpv** [comment]
same exact issue here, haven't found any work around yet.

**Comment by YamiZee** [comment]
Thanks, that worked! Just to make it explicitly clear to anyone else following the guide, run:
`pip install fastapi==0.110 xtts-api-server pydub`

**Comment by YamiZee** [comment]
Thanks, that worked! Just to make it explicitly clear to anyone else following the guide, run:
`pip install fastapi==0.110 xtts-api-server pydub`

**Comment by svlandeg** [comment]
Hi! We've now released spaCy 3.7.5 which should hopefully fix the underlying version conflict 🤞.

**Comment by daswer123** [comment]
Awesome, thanks for your work )
